boards/jiminy-mega256rfr2/Makefile.include

```diff
@@ -0,0 +1,20 @@
+include $(RIOTBOARD)/common/arduino-atmega/Makefile.include
+
+# define the cpu used by the jiminy board
+export CPU = atmega256rfr2
+
+USEMODLUE += periph
+
+# export needed for flash rule
+export PORT_LINUX ?= /dev/ttyACM0
+export PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.usbmodem*)))
+
+# Serial Baud rate is configured to 500kBaud
+# see include/asm-generic/termbits.h for availabel baudrates on your linux system
+export PROGRAMMER_SPEED ?= 0010005
+
```

Comment:
You should add `export BAUD = 38400` somewhere here. This speed is used by RIOTs terminal emulator `pyterm` which is called with `make term`. Without this, a baudrate of `9600` is used which is set in the atmega-common implementation.

Reply:
As i work in eclipse i don't use pyterm.
But i tested it and it works, just the missing/wrong BAUD, and it also works with 250kBaud.